package.json

```diff
@@ -49,8 +49,8 @@
     "stylelint-scss": "^3.21.0",
     "webpack": "^5.71.0",
     "webpack-cli": "^4.9.2",
-    "webpack-dev-server": "^4.8.1",
-    "workbox-webpack-plugin": "^6.3.0"
+    "webpack-dev-server": "^4.7.4",
+    "workbox-webpack-plugin": "^6.5.1"
```

Comment:
- [ ] I am facing this error message from browser as well as terminal when I run live server of your application. I noticed it because I can build the `dist` directory folder even after running `npm install`, `npm run build` and `npm start`. Please fix the problem.

![today1](https://user-images.githubusercontent.com/79058364/163345178-aec4b165-4e41-42a1-9998-4b80a50ba587.png)

Reply:
you can fix by install ''  npm audit fix ''

Reply:
resolved 👍

Reply:
Thank you